src/main/java/org/ajoberstar/gradle/bintray/BintrayRules.java

```diff
@@ -11,35 +26,44 @@
 import org.gradle.api.publish.PublishingExtension;
 
 class BintrayRules extends RuleSource {
-    @Model
-    public void bintray(Bintray bintray) {
-        // just create the model element
-    }
+  @Model
+  public void bintray(Bintray bintray) {
+    // just create the model element
+  }
 
-    @Defaults
-    public void bintrayDefaults(Bintray bintray) {
-        bintray.setPublish(true);
-    }
+  @Defaults
+  public void bintrayDefaults(Bintray bintray) {
+    bintray.setPublish(true);
+  }
 
-    @Validate
-    public void bintrayValidate(Bintray bintray) {
-        Objects.requireNonNull(bintray.getOwner(), "bintray.owner must be populated");
-        Objects.requireNonNull(bintray.getRepo(), "bintray.repo must be populated");
-        Objects.requireNonNull(bintray.getPkg(), "bintray.pkg must be populated");
-    }
+  @Validate
+  public void bintrayValidate(Bintray bintray) {
+    Objects.requireNonNull(bintray.getOwner(), "bintray.owner must be populated");
+    Objects.requireNonNull(bintray.getRepo(), "bintray.repo must be populated");
+    Objects.requireNonNull(bintray.getPkg(), "bintray.pkg must be populated");
+  }
 
-    @Mutate
-    public void addBintrayRepository(PublishingExtension publishing, Bintray bintray) {
-        String url = String.format("https://api.bintray.com/maven/%s/%s/%s/;publish=%s", bintray.getOwner(), bintray.getRepo(), bintray.getPkg(), bintray.getPublish() ? "1" : "0");
-        publishing.repositories(repositories -> {
-            repositories.maven(repo -> {
+  @Mutate
+  public void addBintrayRepository(PublishingExtension publishing, Bintray bintray) {
+    String url =
+        String.format(
+            "https://api.bintray.com/maven/%s/%s/%s/;publish=%s",
+            bintray.getOwner(),
+            bintray.getRepo(),
+            bintray.getPkg(),
+            bintray.getPublish() ? "1" : "0");
+    publishing.repositories(
+        repositories -> {
+          repositories.maven(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 17. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AIndentationCheck)